src/main/java/org/scijava/plugins/scripting/python/OptionsPython.java

```
@@ -145,18 +145,25 @@ public void load() {
 	}
 
 	public void rebuildEnv() {
-		// Use scijava.app.python-env-file system property if present.
+		File environmentYaml = getEnvironmentYamlFile();
```

Should fail fast if the file does not exist?